specs/content-templates.spec.ts

```diff
@@ -10,6 +10,7 @@ test('user input block can trigger content-only block', async ({ page }) => {
   </div>
   <div class="os-raise-ib-content" data-wait-for-event="eventnameX" data-schema-version="1.0">
     <p>Conditional content</p>
+    <p>A sentence with the word <span class="os-raise-ib-tooltip" data-store="glossary-tooltip">absolute value</span> with its definition as a tooltip</p>
```

Comment:
So your test case changes in this file pass, but they're not actually testing whether tooltips work within `os-raise-ib-input` but instead that they work in `os-raise-ib-content`. I suggest that instead of the changes to these tests we add a line like this (and corresponding assertions) to:

* [This](https://github.com/openstax/k12-apps-raise/blob/ca67fcfbaf20dc25f95bc74669183bf80dd6bf9d/specs/blocks.spec.ts#L4) test case
  * Add tooltip content within each section (e.g. `os-raise-ib-input-content`, `os-raise-ib-input-prompt`, and `os-raise-ib-input-ack`)

* [This](https://github.com/openstax/k12-apps-raise/blob/ca67fcfbaf20dc25f95bc74669183bf80dd6bf9d/specs/blocks.spec.ts#L28) test case
    * Add tooltip content within each section (e.g. os-raise-ib-cta-content`, and `os-raise-ib-cta-prompt`)

For :point_up: to pass, you'll need to add invocations to `tooltipify` [here](https://github.com/openstax/k12-apps-raise/blob/ca67fcfbaf20dc25f95bc74669183bf80dd6bf9d/src/components/CTABlock.tsx#L26) and [here](https://github.com/openstax/k12-apps-raise/blob/ca67fcfbaf20dc25f95bc74669183bf80dd6bf9d/src/components/UserInputBlock.tsx#L40).